src/sign-in-windows-forms/MainWindow.cs

```diff
@@ -0,0 +1,116 @@
+using Microsoft.Identity.Client;
+using System.Net.Http.Headers;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+
+namespace MsalExample
+{
+    public partial class MainWindow : Form
+    {
+        private readonly HttpClient httpClient = new();
+
+        // Generally, your MSAL client will have a lifecycle that matches the lifecycle
+        // of the user's session in the application. In this sample, the lifecycle of the
+        // MSAL client to the lifecycle of this form.
+        private readonly IPublicClientApplication msalPublicClientApp;

```

Comment:
```suggestion
        // Unlike a production application, the lifecycle of this code sample's
        // MSAL client instance is that of this form's - the MSAL client is
        // disposed when this form is disposed.
        // In a production desktop application, the user's application session
        // should determine the lifecycle of the MSAL client object. For example,
        // by using a singleton instance of the MSAL client object that you
        // dispose when the user signs out of (or closes) the application.
        private readonly IPublicClientApplication _msalPublicClientApp;
```

This is too much, honestly, but wanted to check with you to see if that's reality and something we can say in the article text. I think our in-code comments will likely be much shorter, but what you and the team are putting in now is great because it'll help us translate into "doc-speak" when we get there.

Reply:
Yup, you picked up what I was putting down.  You explained in a more verbose form what I was trying to imply in my two sentences.

Reply:
@ckittel Thanks, good call-out on the client disposal.

FYI @Dickson-Mwendia